modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -140,7 +161,10 @@ const OrderableTableRow = ({
 				);
 
 				return (
-					<ClayTable.Cell key={field.name}>
+					<ClayTable.Cell
+						headingTitle={field.headingTitle}
```

Comment:
The name of this prop is super-confusing. But, it's like that in Clay so :+1:

Reply:
Ya, not the greatest name, but consistency is good